<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainres.qrc">
    <normaloff>:/icons/icons/metadata-icon-png-8.png</normaloff>:/icons/icons/metadata-icon-png-8.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item row="0" column="2" rowspan="2">
     <widget class="QGroupBox" name="AddMetaGroup">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <spacer name="LeftSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="MidSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QtMaterialFlatButton" name="AddMetaButt">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mainres.qrc">
           <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
         </property>
         <property name="foregroundColor" stdset="0">
          <color>
           <red>78</red>
           <green>154</green>
           <blue>6</blue>
          </color>
         </property>
         <property name="isHaloVisible" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QtMaterialFlatButton" name="DelMetButt">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>25</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mainres.qrc">
           <normaloff>:/icons/icons/clear.png</normaloff>:/icons/icons/clear.png</iconset>
         </property>
         <property name="isHaloVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="foregroundColor" stdset="0">
          <color>
           <red>204</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="RightSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QListView" name="AddedMetaList"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="IOGroup">
      <property name="minimumSize">
       <size>
        <width>561</width>
        <height>131</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>561</width>
        <height>131</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QtMaterialTextField" name="OpenLine">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Select A File To Open</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QtMaterialTextField" name="SaveLine">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Save Path</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" alignment="Qt::AlignHCenter">
        <widget class="QtMaterialFlatButton" name="PurgeMetaButt">
         <property name="minimumSize">
          <size>
           <width>129</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>129</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Purge All Meta</string>
         </property>
         <property name="isHaloVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="foregroundColor" stdset="0">
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
        </widget>
       </item>
       <item row="2" column="0" alignment="Qt::AlignLeft">
        <widget class="QtMaterialFlatButton" name="OpenButt">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
         <property name="foregroundColor" stdset="0">
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
         <property name="isHaloVisible" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2" alignment="Qt::AlignHCenter">
        <widget class="QtMaterialFlatButton" name="SaveButt">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="foregroundColor" stdset="0">
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
         <property name="isHaloVisible" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QtMaterialFlatButton" name="OpenBrowse">
         <property name="minimumSize">
          <size>
           <width>26</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>26</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="isHaloVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="foregroundColor" stdset="0">
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
         <property name="fontSize" stdset="0">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QtMaterialFlatButton" name="SaveBrowse">
         <property name="minimumSize">
          <size>
           <width>26</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>26</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="isHaloVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="foregroundColor" stdset="0">
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
         <property name="fontSize" stdset="0">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QtMaterialFlatButton" name="OverwriteCheck">
         <property name="minimumSize">
          <size>
           <width>129</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>129</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Overwrite</string>
         </property>
         <property name="foregroundColor" stdset="0">
          <color>
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTabWidget" name="ReadMetaWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="GeneralMeta">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtMaterialFlatButton</class>
   <extends>QPushButton</extends>
   <header>qtmaterialflatbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialTextField</class>
   <extends>QLineEdit</extends>
   <header>qtmaterialtextfield.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>OpenLine</tabstop>
  <tabstop>OpenBrowse</tabstop>
  <tabstop>SaveLine</tabstop>
  <tabstop>SaveBrowse</tabstop>
  <tabstop>OpenButt</tabstop>
  <tabstop>PurgeMetaButt</tabstop>
  <tabstop>OverwriteCheck</tabstop>
  <tabstop>SaveButt</tabstop>
  <tabstop>AddMetaButt</tabstop>
  <tabstop>DelMetButt</tabstop>
  <tabstop>AddedMetaList</tabstop>
 </tabstops>
 <resources>
  <include location="mainres.qrc"/>
 </resources>
 <connections/>
</ui>
